packages/core/utils/src/solidity/remapImports.ts

```diff
@@ -72,7 +72,7 @@ const buildNewFile = (file: File, importPath: string) => {
   // imported from node_modules, ie import "@aragon/os/contracts/acl/ACL.sol"
   if (isUnresolvedNodeModule(importPath)) {
     from = resolve(importPath);
-    to = importPath.includes(dappPath(".embark")) ? importPath : dappPath(".embark", "contracts", "node_modules", importPath);
+    to = importPath.includes(dappPath(".embark")) ? importPath : dappPath(".embark", "node_modules", importPath);
```

Comment:
That's something that @michaelsbradleyjr changed last week, not sure if this was a mistake on his part or we're just re-enabling the bug he himself fixed?

Reply:
My bad, it was @emizzle

Reply:
Here is the commit: https://github.com/embark-framework/embark/commit/acd1d72f2d69a006c450428d591677c5a31c6f27